docs/RuntimeAttributes.md

```diff
@@ -27,6 +25,37 @@ workflow jes_workflow {
 }
 ```
 
+
+## Recognized Runtime attributes and Backends
+
+Cromwell recognizes certain runtime attributes and has the ability to format these for some [Backends](/backends/Backends). See the table below for common attributes that apply to _most_ backends.
+
+
+> [Shared Filesystem backend](/backends/HPC#shared-filesystem) (SFS) is fully configurable and thus these attributes do not apply universally.
+
+| Runtime Attribute    | LOCAL |  Google Cloud  |  SGE  | AWS Batch |
+| -------------------- |:-----:|:-----:|:-----:|:------:|
+| [cpu](#cpu)                  |       |   x   |   x   |  x  |
+| [memory](#memory)                              |       |   x   |   x   |  x  |
+| [disks](#disks)                                |       |   x   |       |  x  |
+| [docker](#docker)                              |   x   |   x   |   x   |  x  |
+| [maxRetries](#maxretries)                      |   x   |   x   |   x   |  x  |
+| [continueOnReturnCode](#continueonreturncode) |   x   |   x   |   x   |  x  |
+| [failOnStderr](#failonstderr)                  |   x   |   x   |   x   |  x  |
+
+
+
+### Cloud specific attributes
```

Comment:
If this is only for google cloud attributes, should this section not be titled "Google Cloud Specific Attributes"?

Reply:
We had this discussion on the night in case any other AWS attributes popped up then we wouldn't have lots of `###` level headings. Anyway, I think it makes sense to have a proper subsubsection for each cloud specific, so I changed it to `### Google Cloud ...` in a86b07e248163b124780141b4c89df0d99d21051.